xml/System/TypeInitializationException.xml

```diff
@@ -58,68 +58,68 @@
   
- What often makes this exception difficult to troubleshoot is that static constructors are not always explicitly defined in source code. A static constructor exists in a type if:  
+   What often makes this exception difficult to troubleshoot is that static constructors are not always explicitly defined in source code. A static constructor exists in a type if:  
   
--   It has been explicitly defined as a member of a type.  
+   -   It has been explicitly defined as a member of a type.  
   
--   The type has  `static` (in C#) or `Shared` (in Visual Basic) variables that are declared and initialized in a single statement. In this case, the language compiler generates a static constructor for the type. You can inspect it by using a utility such as [IL Disassembler](~/docs/framework/tools/ildasm-exe-il-disassembler.md). For instance, when the C# and VB compilers compile the following example, they generate the IL for a static constructor that is similar to this:  
+   -   The type has  `static` (in C#) or `Shared` (in Visual Basic) variables that are declared and initialized in a single statement. In this case, the language compiler generates a static constructor for the type. You can inspect it by using a utility such as [IL Disassembler](~/docs/framework/tools/ildasm-exe-il-disassembler.md). For instance, when the C# and VB compilers compile the following example, they generate the IL for a static constructor that is similar to this:  
   
-    ```  
+      ```  
+      .method private specialname rtspecialname static   
+              void  .cctor() cil managed  
+      {  
+        // Code size       12 (0xc)  
+        .maxstack  8  
+        IL_0000:  ldc.i4.3  
+        IL_0001:  newobj     instance void TestClass::.ctor(int32)  
+        IL_0006:  stsfld     class TestClass Example::test  
+        IL_000b:  ret  
+      } // end of method Example::.cctor  
+      ```  
   
-    .method private specialname rtspecialname static   
-            void  .cctor() cil managed  
-    {  
-      // Code size       12 (0xc)  
-      .maxstack  8  
-      IL_0000:  ldc.i4.3  
-      IL_0001:  newobj     instance void TestClass::.ctor(int32)  
-      IL_0006:  stsfld     class TestClass Example::test  
-      IL_000b:  ret  
-    } // end of method Example::.cctor  
+      The following example shows a <xref:System.TypeInitializationException> exception thrown by a compiler-generated static constructor. The `Example` class includes a `static` (in C#) or `Shared` (in Visual Basic) field of type `TestClass` that is instantiated by passing a value of 3 to its class constructor.  That value, however, is illegal; only values of 0 or 1 are permitted. As a result, the `TestClass` class constructor throws an <xref:System.ArgumentOutOfRangeException>. Since this exception is not handled, it is wrapped in a <xref:System.TypeInitializationException> exception.  
```

Comment:
because of the far indentation, this is being interpreted as code and being rendered with the previous code block.

![image](https://user-images.githubusercontent.com/12971179/41893124-9153b8e4-78cf-11e8-9c60-3ec607141736.png)